stdlib/public/TensorFlow/Dataset.swift

```diff
@@ -185,14 +185,27 @@ extension DatasetIterator : IteratorProtocol {
   }
 }
 
+/// A 2-tuple-like struct that conforms to TensorGroup that represents the 
+/// result type of a zip operation between two datasets.
+@_fixed_layout
+public struct ZippedTensors<T : TensorGroup, U : TensorGroup> : TensorGroup {
```

Comment:
I feel the name of this struct should be closer to [`Zip2Sequence`](https://developer.apple.com/documentation/swift/zip2sequence). Perhaps `Zip2Tensors`?

Reply:
`Zip2Tensors` sounds good!